mesonbuild/modules/windows.py

```diff
@@ -75,14 +75,26 @@ def _find_resource_compiler(self, state: 'ModuleState') -> T.Tuple[ExternalProgr
         rescomp = ExternalProgram.from_bin_list(state.environment, for_machine, 'windres')
 
         if not rescomp or not rescomp.found():
+            def search_programs(names: T.List[str]) -> T.Optional[ExternalProgram]:
+                for name in names:
+                    program = ExternalProgram(name, silent=True)
+                    if program.found():
+                        return program
+                return None
+
+            def find_rc_like_compiler() -> T.Optional[ExternalProgram]:
+                return search_programs(['rc', 'llvm-rc'])
```

Comment:
This function exists solely to be an alias for another function. I don't think you need the extra function definition...

Reply:
Changed! :)